### DISCUSSION TITLE
[Config Support]: Procedure to optimize min_score and threshold

### DISCUSSION THREAD

**Discussion by JohannCR** [blakeblackshear/frigate#12823] [Config Support]
### Describe the problem you are having

Hi there, still a fan of frigate after a year playing with its awesomeness, but still trying to fine tune it and finding it kinda hard sometimes even after all this time 😁

I read and reread the docs about min_score and threshold filters, and I think I understand them correctly. But what would be very helpful in the docs is a procedure to help fine tune those filters.

Context:
In my case I'm trying to improve cat and dog detection. Using frigate+ with a model trained a few times with now 100 to 150 verified images per camera.
Most of the time dog and cat detection work great, but there are false positives, mainly when a cat is identified as a dog (dog identified as cat is a lot more rare).
I don't remember what were the min_score and threshold when I generated the images for frigate+, but it was lower than the actual min_score 0.8 and threshold 0.9.

Actual question :
In the docs it says to lower the threshold to get more false positives for the training. So let's say I put the threshold to 0.8. I should probably lower the min_score too, shouldn't I ? To like 0.7 ?
How do we know the min_score is good enough ?
For the threshold it's pretty straightforward, too low leads to more false positives in events, too high leads to less events. Easy to notice !
For the mean score I'd say too low would lead to confused object tracking, so messing up events beginnings and ends, while too high (say equal to the threshold value) would make each detection as a single event (beginning and end) with no tracking at all ? Whether I'm wrong or not, how do we track the effectiveness of our min_score ?

Thanks for any help

### Version

0.13.2-6476f8a go2rtc 1.8.5

### Frigate config file

```yaml
No need here if I'm clear enough in the description ^^
```

### Relevant log output

```shell
None
```

### Operating system

Other Linux

### Install method

Docker Compose

### Object Detector

Coral

**Comment by NickM-27** [comment]
Your min score should not be the same as the threshold. That might explain some of the issues that you are having. 

Min score is mainly used to reduce the tracking effort but too high of a min score will cause true objects to get stuck being traded because they scores lower in a few frames. I'd leave min score at the recommended values (around 0.6)
